frontend/src/metabase/query_builder/components/DownloadWidget.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { Component, PropTypes } from "react";
+import ReactDOM from "react-dom";
+
+import PopoverWithTrigger from "metabase/components/PopoverWithTrigger.jsx";
+import Icon from "metabase/components/Icon.jsx";
+import DownloadButton from "metabase/components/DownloadButton.jsx";
+
+import FieldSet from "metabase/components/FieldSet.jsx";
+
+export default class DownloadWidget extends Component {
```

Comment:
It looks like we aren't needing any of the functions of a full component class here so I wonder if it would make things a bit simpler to just use a stateless functional component here?